### PR TITLE
Feature/project card improvemetns

### PR DESCRIPTION
**Pull request by hkurra**
Project cards Improvements
NDVI improvements
User contribution Implementation

### CODE REVIEW

**Comment by hkurra** [comment]
@norbertschuler @sagararyal I request to merge this PR. Both of you guys are aware how Agile works. There are no changes in this PR which are breaking previous implementations.
I have worked on this for quite a long time and I don't think there should be a bing bang merge. If the feature is partially complete and not breaking the current flow, I seriously think you folks should merge this. We shouldn't hold this PR for long. That's not how Agile works. If you have any compelling reasons to block it then please let know.

**Comment by norbertschuler** [comment]
@hkurra thanks for explaining. Has anyone else but you reviewed and tested this code yet? If we merge this code and there are issues we just would need your full commitment to quickly fix all raised issues. If you want me to first do a review I am afraid I can not do it before Monday.

**Comment by sagararyal** [comment]
@Hkurra I am blocking the PR until it is feature complete. The features on the requirements would be shipped only when complete.

File: app/components/UserContributions/userContribution.native.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import PropTypes from 'prop-types';
+// import i18n from '../../locales/i18n.js';
+import { Text, View, TouchableOpacity, Image } from 'react-native';
+import styles from '../../styles/newUserContributions/userContributions';
+
+import EditIcon from '../../assets/images/pencil.png';
+import DeleteIcon from '../../assets/images/baseline_delete_outline.png';
+import ArrowRight from '../../assets/images/arrow-right.png';
+import CalendarIcon from '../../assets/images/green-calendar.png';
+import TreeIcon from '../../assets/images/green-tree.png';
```
Comment (norbertschuler) [inline app/components/UserContributions/userContribution.native.js]: Why don't you load the images with app/assets/index.js?

File: ios/Podfile [hunk 2]
```diff
@@ -3,4 +3,6 @@
 
 target 'TreecounterApp' do
   inherit! :search_paths
+  pod 'RNVectorIcons', :path => '../node_modules/react-native-vector-icons'
```
Comment (norbertschuler) [inline ios/Podfile]: I intentionally removed all Pods from here and added the node modules manually to our Xcode project, so (1) navigate to the folder with all the node projects in Xcode and add this project as well there and (2) add any library of this node project in the General tab to the our app. Please consult the documentation of other node modules where this kind of adding a project without using Pods is described.

File: app/components/Measurements/Alert.native.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../../styles/mesurements/alert';
+import { View, Text } from 'react-native';
+
+const Alert = props => (
+  <View style={{ alignItems: 'center' }}>
+    <View style={styles.container}>
+      <Text style={styles.text}>
+        {props.deletedTreeCount ? props.deletedTreeCount : 'None'} Tree
```
Comment (norbertschuler) [inline app/components/Measurements/Alert.native.js]: Please add as translation resource.

File: app/components/Measurements/Alert.native.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        contribution has been deleted
```
Comment (norbertschuler) [inline app/components/Measurements/Alert.native.js]: Please add as translation resource.

File: app/components/Measurements/Alert.native.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      </Text>
+      <Text style={styles.text}>Successfully. redirect to Me page.</Text>
```
Comment (norbertschuler) [inline app/components/Measurements/Alert.native.js]: Please add as translation resource.

File: app/components/Measurements/mockData.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+//planted Date, Height of tree, width of tree
+const data = [
+  { plantedDate: 'On April 9,2019', heightOfTree: 150, widthOfTree: 120 },
+  { plantedDate: 'On January 12,2019', heightOfTree: 123, widthOfTree: 39 },
+  { plantedDate: 'On September 10,2019', heightOfTree: 131, widthOfTree: 42 }
```
Comment (norbertschuler) [inline app/components/Measurements/mockData.js]: Are these data sets used or displayed to the user? If yes, please add as translation resource.

File: package.json [hunk 7]
```diff
@@ -75,6 +75,7 @@
     "react-native-svg": "^6.5.3",
     "react-native-svg-uri": "^1.2.3",
     "react-native-tab-view": "^1.0.2",
+    "react-native-vector-icons": "^6.6.0",
```
Comment (norbertschuler) [inline package.json]: You are adding fonts of 2MB to the iOS and Android project (increasing the app size from 10MB to 12MB). That's not that bad, but anyway I wanted to ask if all these fonts are necessary to make this patch work and what exactly are they used for?

I only found two import as `import Icon from 'react-native-vector-icons/FontAwesome` using just one these fonts to display a star and a flag icon.

I have not yet tested how much KB this module will add to our web clients bundle.js - I will do at Monday (or you can test yourself by building the bundle file with `npm run build:server` and then look at it with `npm run analyze`.

File: app/components/Measurements/mockData.js (hunk 6 above)
Comment (hkurra) [reply app/components/Measurements/mockData.js]: NO, these are mock data

File: package.json (hunk 7 above)
Comment (norbertschuler) [reply package.json]: Maybe it's possible to only add the font `FontAwesome` for now?

File: app/components/Measurements/Alert.native.js [hunk 8]
```diff
@@ -1,22 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import styles from '../../styles/mesurements/alert';
```
Comment (norbertschuler) [inline app/components/Measurements/Alert.native.js]: You have not removed app/styles/mesurements/alert.js

File: ios/TreecounterApp/Info.plist [hunk 9]
```diff
@@ -78,22 +78,11 @@
 		<string>OpenSans-RegularItalic.ttf</string>
 		<string>OpenSans-SemiBold.ttf</string>
 		<string>OpenSans-SemiBoldItalic.ttf</string>
-		<string>AntDesign.ttf</string>
```
Comment (norbertschuler) [inline ios/TreecounterApp/Info.plist]: @hkurra thanks for removing this, but I think that's not enough to the Xcode project to do (see my other comments above) - do you have a MacBook with Xcode and can modify the iOS projects with Xcode?

**Pull request closed without merging**